lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/Cli.scala

```diff
@@ -137,6 +138,12 @@ object Cli {
       .longOpt(FILESYSTEM_WRITE_PATH)
       .desc("Write data from stdin to the provided file")
       .build()
+
+    val attachDebugger: cli.Option = cli.Option.builder
+      .hasArg(false)
+      .longOpt(ATTACH_DEBUGGER)
+      .desc("Enables remote debugging capabilities")
```

Comment:
This isn't sufficient documentation for someone seeing output of `project-manager --help`.